index.js

```diff
@@ -1,11 +1,6 @@
 'use strict';
 const {builtinModules} = require('module');
 
-const ignoreList = [
-	'sys'
-];
```

Comment:
I prefer to keep the ignore list as it's easier to add entries than a regex.

The existing regex could ideally be converted to a list like `ignorePrefixList`.

Reply:
Yeah I thought about `ignorePrefixList`, I was wondering which one you'd prefer, I guessed incorrectly, will update :)